portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -468,150 +578,19 @@ definition {
 			Navigator.gotoNavItem(navItem = "Scopes");
 		}
 
-		task ("Expand global panels in global scopes") {
-			Navigator.gotoNavUnderline(navUnderline = "Global scopes");
-
-			for (var globalPanel : list "analytics.read,analytics.write,everything.write,everything.read,everything") {
-				OAuth2.expandGlobalPanel(globalPanel = "${globalPanel}");
-			}
+		task ("Assert global scopes present") {
```

Comment:
Global scope is what? tab item, page item? I think e should mention where the item type to help identify it better.